ci/goreleaser/goreleaser.yml

```diff
@@ -36,6 +28,7 @@ builds:
       - linux
     goarch:
       - arm64
+    binary: tyk
   # static builds strip symbols and do not allow plugins
```

Comment:
Shouldn't this build be removed, since it used only in the slim images? There are other references to this in other places too.

Reply:
the binary field is for the standard build binary name - since portal uses a separate binary name, there's a separate binary key in the conf for portal. To make it standard across all confs - we've added it and it's the same as the repo name for all  repos except portal. The `static-amd64` build should be removed since slim images are no longer generated.  Missed removing it. I'll fix it.